packages/java/tests/spring/react-signals/package.json

```
@@ -12,6 +12,7 @@
     "@vaadin/hilla-react-auth": "file:../../../../ts/react-auth",
     "@vaadin/hilla-react-crud": "file:../../../../ts/react-crud",
     "@vaadin/hilla-react-form": "file:../../../../ts/react-form",
+    "@vaadin/hilla-react-i18n": "24.4.0-alpha13",
```

Should be a symlink also:
```suggestion
    "@vaadin/hilla-react-i18n": "file:../../../../ts/react-i18n",
```